moveit_setup_assistant/src/widgets/collision_matrix_model.h

```diff
@@ -0,0 +1,80 @@
+/*********************************************************************
```

Comment:
if this is not a widget, i.e. not a view, then I think it belongs in ``src/tools`` next to ``compute_default_collisions.cpp``